test/utils/test.utils.ts

```diff
@@ -1,5 +1,5 @@
 export function wordCount(wordToCount: string, inText: string): number {
-  const regex = new RegExp(`\\b${wordToCount}\\b`, 'gi');
+  const regex = new RegExp(`${wordToCount}`, 'gi');
```

Comment:
Just curious as to what the \\b s did and why they were removed?

Reply:
something to do with word boundaries but it prevented using the function to search for more than one word